themes/ddbasic/sass/components/node/paragraphs.scss

```diff
@@ -0,0 +1,171 @@
+//
+// Styling of paragraphs
+
+@import '../../base.scss';
+
+// Paragraphs block display
+.paragraphs-block {
+  position: relative;
+  display: inline-block;
+  margin: 0 0 $distance-paragraph;
+  width: 100%;
+
+  &.half_left {
+    float: left;
+    padding-right: 3%;
+    width: 50%;
+
+    @include media($notebook) {
+      width: 100%;
+    }
+
+    @include media($tablet) {
+      width: 100%;
+    }
+
+    @include media($mobile) {
+      width: 100%;
+    }
+  }
+
+  &.half_right {
+    float: right;
+    padding-left: 3%;
+    width: 50%;
+
+    @include media($notebook) {
+      width: 100%;
+    }
+
+    @include media($tablet) {
+      width: 100%;
+    }
+
+    @include media($mobile) {
+      width: 100%;
+    }
+  }
+
+  h2 {
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  p:last-child {
+    margin: 0;
+  }
+
+  .inner {
+    cursor: auto;
+    width: 100%;
+    float: left;
+    padding: 15px;
+    box-sizing: border-box;
+    background-color: $grey;
+    word-break: break-all; // IE - We want to break words
+    word-break: break-word; // Override for webkit (It's better)
+
+    .paragraphs-text {
+      margin: 10px 0;
+    }
+
+    .object-left {
+      float: left;
+      width: 40%;
+      margin-right: 5%;
+    }
+
+    .object-right {
+      float: left;
+      margin: 0;
+      width: 55%;
+    }
+  }
+
+  .ting-object {
+    @include display(flex);
+    
+    .ting-object-left {
+      margin-right: 2%;
+      @include flex(0 1 auto);
+
+      .ting-cover > img {
+        min-width: 50px;
+        width: 100%;
+      }
+    }
+
+    .ting-object-right {
+      @include display(flex);
+      @include flex-wrap(wrap);
+      @include flex(1 1 auto);
+
+      .field-name-ting-type {
+        display: none;
+      }
+
+      .field-name-ting-title h2 {
+        margin: 0;
+        -ms-word-break: break-all;
+        word-break: break-all;
+
+        /* Non standard for WebKit */
+        word-break: break-word;
+
+        -webkit-hyphens: auto;
+        -moz-hyphens: auto;
+        hyphens: auto;
+
+        a {
+          color: $color-text;
+
+          &:hover {
+            color: $color-text-link;
+          }
+        }
+      }
+
+      .field-name-ting-abstract {
+        margin: 10px 0;
+        max-width: 100%;
+      }
+
+      .button-see-online,
+      .reserve-button {
+        @include order(10);
+      }
+
+      .action-button {
+        padding: 10px 60px 8px 15px;
+        margin-top: 10px;
+        max-width: 100%;
+      }
+    }
+  }
+
+  &.carousel,
+  &.materials-list {
+    width: 150%;
```

Comment:
This adds a width larger than 100%. Why is this? That seems odd and as far as I can tell it is not a pattern used elsewhere. I cannot tell from code or commit comments.

Reply:
Added comment explaining the need for extra width https://github.com/ding2/ding2/pull/1406/commits/de4a5c8a39cb59c34dfc5bc6c160d4f3702567d7

Again it has to do with the limited content width clashing with paragraphs